### DISCUSSION TITLE
Welcome to pgCenter Discussions!

### DISCUSSION THREAD

**Discussion by lesovsky** [lesovsky/pgcenter#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ishibaev** [comment]
Hello! pgcenter v0.9.2 
Please add ability to exclude columns from activity/statements/etc realtime output
For example -  

- at one time for activity I prefer to see only query & query_age
- next time - only query_age and usename

**Reply by lesovsky** [reply]
Hi,
hmm, it is not clear for me what problem this feature should solve.

**Reply by ishibaev** [reply]
Hi!
let me be clean - at the current moment it's possible to make columns wider OR narrow. and if you need more attention on some columns values vs others, then may be you prefer to extend it (to see whole values without cut), and hide others ( to exclude it from your point of view)
 but here some limitations -
- you can't change column order for view. to see all columns at the terminal window it must be on wide monitor, or some columns will be outside of the screen (for ex, query column is the last)
- columns cant be made with width equal 0 or 1 char
So, you can't hide some columns or change column order (again ex, the query column is the last )
and, because sometimes  not all columns you need to view, ability to temporal exclude unnecessary columns from view will be useful

**Reply by lesovsky** [reply]
ok, sounds clear. what about workflow of the feature? What user have to do for hiding and then for showing the particular column(s)? Press a special hotkey or maybe invoke a menu and select (or unselect) interesting columns?

**Reply by ishibaev** [reply]
it's looks like a lot of different hotkeys already exists, and drop down menu with check boxes will be more useful/prefered
but, I'm not sure is it will be easy to implement it, so may be use 'less change more profit'? for example here  - 
you  have **Left** and **Right**  to change column  selection for sort acs/desc
May be add Shift + Left/Right  and it will can change column position?
this will give 
1. option to move necessary interesting columns to the left  part
2. manually extend moved columns to necessary size
of course, in case when you need to see whole query field, it wouldn't be too easy to do so, but ... to see whole query better do another improvement ( you have G key for pg_stat_statements, and for activity view G key can just show whole query text, for ex)

**Reply by lesovsky** [reply]
Both ways are not easy to implement, but  as an end-user I'd prefer a drop-down menu, it is obvious and simple.

**Comment by lazarevmk** [comment]
Добрый день, Алексей.

Проект больше не развивается или он просто "переехал" куда-то и просто так уже не будет доступен?

**Comment by lesovsky** [comment]
@lazarevmk Проект не развивается, у меня нет на это времени. Есть план перенести разработку в компанию где я сейчас работаю и продолжить, но это примерно только начало следующего года.
